perun-wui-core/src/main/java/cz/metacentrum/perun/wui/client/resources/PerunConfiguration.java

```diff
@@ -706,6 +706,15 @@ public static boolean isWayfLinkAnAccountDisabled() {
 		return attributes;
 	}
 
+	/**
+	 * Checks whether the functionality of finding similar users is disabled.
+	 *
+	 * @return true when the property registrar.findSimilarUsers.disabled is set to true. False otherwise.
+	 */
+	public static boolean findSimilarUsersDisabled() {
```

Comment:
Nit: When defining configuration options, I would prefer using a positive version like `findSimilarUsers`. I think it is more clear than thinking about single and double negations. But this is not really necessary to fix.

Reply:
I have chosen to use negation because of the implementation of method `getConfigPropertyBoolean`. It returns false when the property is not set or it is filled by anything except 'true' (ignoring case).